doc/make.jl

```diff
@@ -0,0 +1,33 @@
+using Documenter, Clustering
+
+makedocs(
+    source = "source",
+    format = Documenter.HTML(prettyurls=false),
+    sitename = "Clustering.jl",
+    modules = [Clustering],
+    pages = [
+        "Clustering.jl Docs" => "index.md",
+        "Algorithms" => [
+            "algorithms.md",
+            "init.md",
+            "kmeans.md",
+            "kmedoids.md",
+            "hclust.md",
+            "mcl.md",
+            "affprop.md",
+            "dbscan.md",
+            "fuzzycmeans.md",
+        ],
+        "Evaluation & Validation" => ["validate.md",
+            "silhouette.md",
+            "randindex.md",
+            "varinfo.md",
+            "vmeasure.md",
+        ]
+    ],
+    debug = false,
+)
+
+deploydocs(
+    repo   = "github.com/alyst/Clustering.jl.git",
```

Comment:
```suggestion
    repo   = "github.com/JuliaStats/Clustering.jl.git",
```

Reply:
Still needs this change, other than that looks good to me

Reply:
Sure. I'll update it at the very last moment :)